server/routes/auth/index.js

```diff
@@ -75,6 +81,7 @@ router.post("/login", async (req, res, next) => {
 });
 
 router.delete("/logout", (req, res, next) => {
+  res.clearCookie("token");
```

Comment:
The [documentation for `res.clearCookie`](http://expressjs.com/en/api.html#res.clearCookie) says that compliant browsers/clients will only clear the cookie if the provided `options` are identical to those given when setting the cookie with `res.cookie()` (aside from `expires` and `maxAge`).